tests/visualizations_tests/test_vis_instance_segmentation.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+import unittest
+
+from chainer import testing
+
+from chainercv.utils import generate_random_bbox
+from chainercv.visualizations import vis_instance_segmentation
+
+try:
+    import matplotlib  # NOQA
+    optional_modules = True
+except ImportError:
+    optional_modules = False
+
+
+@testing.parameterize(
+    {
+        'n_bbox': 3, 'label': (0, 1, 2), 'score': (0, 0.5, 1),
+        'label_names': ('c0', 'c1', 'c2')},
+    {
+        'n_bbox': 3, 'label': (0, 1, 2), 'score': None,
+        'label_names': ('c0', 'c1', 'c2')},
+    {
+        'n_bbox': 3, 'label': (0, 1, 2), 'score': (0, 0.5, 1),
+        'label_names': None},
+    {
+        'n_bbox': 3, 'label': None, 'score': (0, 0.5, 1),
+        'label_names': ('c0', 'c1', 'c2')},
+    {
+        'n_bbox': 3, 'label': None, 'score': (0, 0.5, 1),
+        'label_names': None},
+    {
+        'n_bbox': 3, 'label': None, 'score': None,
+        'label_names': None},
+    {
+        'n_bbox': 3, 'label': (0, 1, 1), 'score': (0, 0.5, 1),
+        'label_names': ('c0', 'c1', 'c2')},
+    {
+        'n_bbox': 0, 'label': (), 'score': (),
+        'label_names': ('c0', 'c1', 'c2')},
+)
+class TestVisInstanceSegmentation(unittest.TestCase):
+
+    def setUp(self):
+        self.img = np.random.randint(0, 255, size=(3, 32, 48))
+        self.mask = np.random.randint(0, 1, size=(self.n_bbox, 32, 48))
```

Comment:
This is always 0.
`np.random.randint(0, 2, size= ..., dtype=np.bool)` is better.